simulation/ESC.h

```diff
@@ -2,12 +2,13 @@
 #define KICKCAT_SIMULATION_ESC_H
 
 #include "kickcat/protocol.h"
-#include <vector>
+#include "kickcat/AbstractESC.h"
+
 #include <functional>
 
 namespace kickcat
 {
-    class ESC
+    class ESCEmulator final : public AbstractESC
```

Comment:
Might as well rename the file to be consistent

Reply:
Right